<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345435</width>
    <height>543455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3498db; /* Цвет фона */
    color: white; /* Цвет текста */
    border-radius: 5px; /* Закругление углов */
 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #78B6FA, stop: 1 #6699CC); 
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Низкополигональный 
фильтр</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Глобальная
 пороговая обработка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Адаптивная
 пороговая обработка</string>
    </property>
   </widget>
   <widget class="QLabel" name="renderingLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="OrigLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498db; /* Цвет фона */
    color: white; /* Цвет текста */
    font-size: 18px; /* Размер шрифта */
    font-weight: bold; /* Жирный шрифт */
    padding: 10px; /* Внутренний отступ */
    border-radius: 5px; /* Закругление углов */</string>
    </property>
    <property name="text">
     <string>Rendering Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498db; /* Цвет фона */
    color: white; /* Цвет текста */
    font-size: 18px; /* Размер шрифта */
    font-weight: bold; /* Жирный шрифт */
    padding: 10px; /* Внутренний отступ */
    border-radius: 5px; /* Закругление углов */</string>
    </property>
    <property name="text">
     <string>Original Label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Откатить к исходному состоянию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Гистограмное
 выравнивание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayParts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50;
    background-color: white;
    color: black;
    padding: 8px 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px; 
    margin: 4px 2px;
    cursor: pointer;
    border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Преобразование
 в оттенки серого</string>
    </property>
   </widget>
   <widget class="QListWidget" name="picList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    color: white; /* Яркий цвет текста */
    font-size: 16px; /* Размер шрифта */
    border: 2px solid #3498db; /* Обводка */
    border-radius: 5px; /* Закругление углов */
}

QListWidget::item {
    background-color: #3498db; /* Цвет фона для каждого элемента */
    padding: 8px; /* Внутренний отступ */
}

QListWidget::item:hover {
    background-color: #2980b9; /* Изменение цвета фона при наведении */
}

QListWidget::item:selected {
    background-color: #2c3e50; /* Цвет фона для выбранного элемента */
    color: white; /* Цвет текста для выбранного элемента */
}

QListWidget::item:selected:hover {
    background-color: #1f2b38; /* Изменение цвета фона выбранного элемента при наведении */
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
